editor/src/core/performance/performance-regression-tests.spec.tsx

```diff
@@ -117,7 +117,7 @@ describe('React Render Count Tests - ', () => {
     )
 
     const renderCountAfter = renderResult.getNumberOfRenders()
-    expect(renderCountAfter - renderCountBefore).toBeGreaterThanOrEqual(460) // if this breaks, GREAT NEWS but update the test please :)
-    expect(renderCountAfter - renderCountBefore).toBeLessThan(470)
+    expect(renderCountAfter - renderCountBefore).toBeGreaterThanOrEqual(620) // if this breaks, GREAT NEWS but update the test please :)
```

Comment:
I'm guessing this just means it's definitely forced at this stage when in all likelihood otherwise it would trigger in the next couple of frame anyway?

Reply:
exactly. and we actually prefer the slow update to happen when you change the selected view, instead of some random action triggering it later